### ISSUE TITLE
PROBLEMS tab shows include errors

### ISSUE THREAD

**Issue by finikorg** [KozhinovAlexander/zephyr_vscode_workspace#3]
I am able to compile project with `west build` task but I assume that vscode uses some other build configuration to detect errors, like include, etc.

For the simplest hello_world app it shows errors like:

```
#include errors detected based on information provided by the configurationProvider setting. Squiggles are disabled for this translation unit (<mypath>/zephyrproject/zephyr/samples/hello_world/src/main.c).

cannot open source file "zephyr/zephyr.h"
```

Do I need to disable this smart analyzer or change build config?

**Comment by KozhinovAlexander** [comment]
Hello,

Thank you for using my work. I hope it will help you to go forward and be productive.

Well, this kind of errors were expected from me. The reason for that is currently missed proper configuration for Zephyr's binding into workspace. Thus please ignore it. I am using C++ Intellisense extension in VSCode for code navigation. It seems to be enough. Therefore I had no need to provide proper build configuration.

Please do not hesitate to provide a pull request if you'd like to solve this issue. I will test it and try give it a way.

Best regards,
Alexander Kozhinov

**Comment by KozhinovAlexander** [comment]
Hello @finikorg Your problem should be fixed by PR #4  now. Please consider it works for you or point on problems you're still facing. :)

**Comment by KozhinovAlexander** [comment]
Approved to work
